src/client/containers/BoardPageComponents/BoardAddCandidate/AddCandidate.js

```diff
@@ -0,0 +1,43 @@
+// import React, { useState } from 'react';
+import postData from './PostData';
+
+export default async function AddCandidate(
+  newCandidate,
+  userId,
+  boardId,
+  setCandidates,
+  candidates,
+  setAddCandidateError,
```

Comment:
Please remove all erorr handling. We will add this later

Reply:
Done